README.md

```diff
@@ -142,12 +142,10 @@ standards (no plugins, just HTML5 and WebRTC)! It's easy to get started!
 ##### Downloading a file is simple:
 
 ```js
-var WebTorrent = require('webtorrent')
-
 var client = new WebTorrent()
-var magnetURI = '...'
+var torrentId = 'magnet:?xt=urn:btih:6a9759bffd5c0af65319979fb7832189f4f3c35d&dn=sintel.mp4&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&tr=wss%3A%2F%2Ftracker.webtorrent.io&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel-1024-surround.mp4'
```

Comment:
@DiegoRBaquero

Some new people who absolutely unfamiliar with WebTorrent might think that magnet URI is the only way to specify torrent in this example.

I've changed the name of variable `magnetURI` to `torrentId` to emphasize that users can use not only magnet URIs.

`torrentId` is more universal variable name than `magnetURI`.